<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiagnosticYearReportSetupDialog</class>
 <widget class="QDialog" name="DiagnosticYearReportSetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Годовой отчет по диагностическим отделениям</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QCheckBox" name="chkUrgent">
     <property name="text">
      <string>Срочные</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblAgeFrom">
     <property name="text">
      <string>Возраст с</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <widget class="CPersonComboBoxEx" name="cmbPerson"/>
   </item>
   <item row="3" column="5">
    <widget class="QLabel" name="lblAgeYears">
     <property name="text">
      <string>лет</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblReportYear">
     <property name="text">
      <string>Отчётный год</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblPerson">
     <property name="text">
      <string>Исполнитель</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QSpinBox" name="edtReportYear">
     <property name="minimum">
      <number>2000</number>
     </property>
     <property name="maximum">
      <number>2500</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>Назначившее отделение</string>
     </property>
     <property name="buddy">
      <cstring>cmbOrgStructure</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="5">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="7" column="2" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QSpinBox" name="edtAgeTo">
     <property name="maximum">
      <number>150</number>
     </property>
     <property name="value">
      <number>150</number>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="lblAgeTo">
     <property name="text">
      <string>по</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QSpinBox" name="edtAgeFrom">
     <property name="maximum">
      <number>150</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>129</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtReportYear</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DiagnosticYearReportSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DiagnosticYearReportSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
